src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -253,7 +253,35 @@ public override uint GetChildCount()
         /// <since_tizen> 4 </since_tizen>
         public override Container GetParent()
         {
-            return InternalParent as Container;
+            if (InternalParent == null && !(this is Window))
+            {
+                // Ask to dali, recersively.
+                
+                IntPtr cPtrParent = Interop.Actor.GetParent(SwigCPtr);
+                HandleRef parent = new global::System.Runtime.InteropServices.HandleRef(this, cPtrParent);
+                while (Interop.BaseHandle.HasBody(parent) && (Registry.GetManagedBaseHandleFromNativePtr(parent.Handle) == null))
+                {
+                    cPtrParent = Interop.Actor.GetParent(parent);
+
+                    Interop.BaseHandle.DeleteBaseHandle(parent);
+                    parent = new global::System.Runtime.InteropServices.HandleRef(this, cPtrParent);
+                }
+
+                if (!Interop.BaseHandle.HasBody(parent))
+                {
+                    Interop.BaseHandle.DeleteBaseHandle(parent);
+                    return null;
+                }
+                
+                BaseHandle baseHandle = Registry.GetManagedBaseHandleFromNativePtr(parent.Handle);
+                InternalParent = baseHandle;
+
+                return baseHandle as Container;
```

Comment:
We may need to call `DeleteBaseHandle` for `cPtrParent` before return it. See how `Tizen.NUI.Extensions.GetInstanceSafety<>` implemented.

```suggestion
                Interop.BaseHandle.DeleteBaseHandle(parent);
                parenr = new HandelRef(null, IntPtr.Zero);
                return baseHandle as Container;
```

Reply:
Thanks for your comment. I think we forgot to Delete(parent). But I wonder how the last line(:280) matters. Why would you put another new (zero pointer handle) to this local variable (parent)?

I tried to search Tizen.NUI.Extensions.GetInstanceSafety<> function but could not find it.